Modules/FindCUDA.cmake

```diff
@@ -41,6 +40,17 @@
 #      nvcc in the generated source.  If you compile to PTX and then load the
 #      file yourself, you can mix bit sizes between device and host.
 #
+#   CUDA_ARCH_NAME (Default is "Auto" for regular builds and "Manual" for cross-builds)
```

Comment:
I would prefer this being named something like CUDA_ARCH_TARGETS or CUDA_BUILD_TARGETS or something besides "NAME".  This isn't a name.  This is a variable that drives which architecture targets are produced by CUDA_SELECT_NVCC_ARCH_FLAGS.  I would even prefer it be named something to do with the name of the function that actually cares about it, but perhaps that would be too wordy.